code/html assignment1/styles.css

```diff
@@ -0,0 +1,3 @@
+.main{padding: 20px;margin: auto;font-family: 'Segoe UI', sans-serif;}
```

Comment:
Why is your CSS like this? Why are you writing it all on one line? Like it technically works but it is extremely hard to read. Do you actually write it like this?